tests/components/dynalite/test_config_flow.py

```diff
@@ -1,36 +1,43 @@
 """Test Dynalite config flow."""
-from unittest.mock import Mock, call, patch
-
-from homeassistant.components.dynalite import config_flow
-
-from tests.common import mock_coro
-
-
-async def test_step_import():
-    """Test a successful setup."""
-    flow_handler = config_flow.DynaliteFlowHandler()
-    with patch.object(flow_handler, "context", create=True):
-        with patch.object(flow_handler, "hass", create=True) as mock_hass:
-            with patch.object(
-                flow_handler, "async_create_entry", create=True
-            ) as mock_create:
-                host = "1.2.3.4"
-                entry1 = Mock()
-                entry1.data = {"host": host}
-                entry2 = Mock()
-                entry2.data = {"host": "5.5"}
-                mock_hass.config_entries.async_entries = Mock(
-                    return_value=[entry1, entry2]
-                )
-                mock_hass.config_entries.async_remove = Mock(
-                    return_value=mock_coro(Mock())
-                )
-                await flow_handler.async_step_import({"host": "1.2.3.4"})
-                mock_hass.config_entries.async_remove.assert_called_once()
-                assert mock_hass.config_entries.async_remove.mock_calls[0] == call(
-                    entry1.entry_id
-                )
-                mock_create.assert_called_once()
-                assert mock_create.mock_calls[0] == call(
-                    title=host, data={"host": host}
-                )
+from asynctest import patch
+
+from homeassistant import config_entries
+from homeassistant.components import dynalite
+
+
+async def run_flow(hass, setup, connection):
+    """Run a flow with or without errors and return result."""
+    host = "1.2.3.4"
+    hass.data[dynalite.DOMAIN] = {}
+    hass.data[dynalite.DOMAIN][dynalite.DATA_CONFIGS] = {}
+    hass.data[dynalite.DOMAIN][dynalite.DATA_CONFIGS][host] = {dynalite.CONF_HOST: host}
+    with patch.object(dynalite.DynaliteBridge, "async_setup", return_value=setup):
+        with patch.object(
+            dynalite.DynaliteBridge, "try_connection", return_value=connection
```

Comment:
Please patch the library interface. Don't patch our code.

Reply:
can you please explain? Isn't DynaliteBridge the library?

Reply:
some additional context - DynaliteBridge is not an Entity in hass
Anyway, please explain and i will fix

Reply:
ok. i think i understand what you want. however, i still have a problem. 

i can patch the library so it will return connected or disconnected (like in test_bridge.py). However, if it is disconnected, it will wait 30 seconds to connect, which we don't want in unit-tests. I implemented it and will push shortly, but it still has to patch bridge constants so it won't wait for the full timeout.

Reply:
sorry. i thought i had it figured out. the problem is that i need to "edit" the configuration.yaml file so I can supply the config to the component otherwise the bridge doesn't set up. Is there a way to change the config of the hass fixture in the tests?

Reply:
It's ok to patch the reconnection time constants in the bridge module.

Reply:
ok. fixed. thanks for the help!